src/js/components/AppListComponent.jsx

```diff
@@ -186,7 +187,10 @@ var AppListComponent = React.createClass({
 
     if (filterText != null && filterText !== "") {
       nodesSequence = nodesSequence
-        .filter(app => app.id.indexOf(filterText) !== -1);
+        .filter(app => {
+          return score(app.id, filterText) > 0.025;
```

Comment:
OK, now I got it. You want fuzzy matched results, besides direct matches.
This makes sense.
I was confused because this behavior isn't reflect in the related issue.
It only says that the given results should be _ranked_ better. That's how I understand it.

With this, now, you modify the complete result set.
But I like it! 

Confirmative. :)